data/mainnet/tokens.js

```diff
@@ -83,6 +83,26 @@ module.exports = {
     cmcRewardTokenSymbols: ['iFARM', 'DON', 'WETH'],
     disableAutoSwap: true,
   },
+  crvIbEUR: {
+    isNew: true,
+    chain: CHAINS_ID.ETH_MAINNET,
+    category: VAULT_CATEGORIES_IDS.STABLECOINS,
+    displayName: 'CRV:ibEUR',
+    apyIconUrls: ['./icons/curve-ibeur.png', './icons/cvx.png', './icons/rKP3R.png'],
```

Comment:
The first icon should be for CRV token, not the curve-ibeur token.

Reply:
done